Enterprise-CMCS/cmcs-eregulations
# Update sass

Resolves Snyk finding

**Description**

This pull request recreates the [Snyk PR seen here](https://github.com/Enterprise-CMCS/cmcs-eregulations/pull/1253) that will not deploy correctly.

**This pull request changes:**

- Updates sass to 1.72.0

**Steps to manually verify this change:**

1. Green check marks
2. [Experimental deployment](https://ya1aelbyne.execute-api.us-east-1.amazonaws.com/dev1264) looks and feels as expected with no stylistic differences compared to `prod`
3. Site builds and can be watched locally without errors or issues
